adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
How could that cause division by zero..? It's the same operation, except that you don't have floating point inaccuracy (which does matter as the timestamp gets larger).